recipes/AISHELL-1/NAR-ASR/NAR_BERT_ASR/hparams/pretrain_encoder.yaml

```diff
@@ -0,0 +1,194 @@
+# ############################################################################
+# Model: NAR-BERT-ASR-Encoder
+# https://arxiv.org/pdf/2104.04805.pdf
+# ############################################################################
+# Seed needs to be set at top of yaml, before objects with parameters are made
+seed: 8886
+__set_seed: !apply:torch.manual_seed [!ref <seed>]
+output_folder: results/Pretrain_Encoder
+cer_file: !ref <output_folder>/cer.txt
+save_folder: !ref <output_folder>/save
+train_log: !ref <output_folder>/train_log.txt
+
+# Data files
+data_folder: !PLACEHOLDER # e,g./path/to/aishell
+# noise/ris dataset will automatically be downloaded
+# data_folder_rirs: !ref <data_folder> # Change this is needed
+skip_prep: False
+ckpt_interval_minutes: 15 # save checkpoint every N min
+train_data: !ref <output_folder>/train.csv
+valid_data: !ref <output_folder>/dev.csv
+test_data: !ref <output_folder>/test.csv
+
+# Training parameters
+number_of_epochs: 130
+batch_size: 14
+gradient_accumulation: 12
+gradient_clipping: 5.0
+loss_reduction: 'mean'
+sorting: random
+
+# stages related parameters
+stage_one_epochs: 130
+lr_adam: 1.0
+lr_sgd: 0.000025
+
+# Feature parameters
+sample_rate: 16000
+n_fft: 400
+n_mels: 80
+
+# maximum decoding length
```

Comment:
Expressed in terms of what ? I think a bit more of details are required to explain what is this maximum_length used for please.